src/compile/scale/range.ts

```diff
@@ -226,11 +226,10 @@ export function defaultRange(
         // Only nominal data uses ordinal scale by default
         return type === 'nominal' ? 'category' : 'ordinal';
       } else if (isContinuousToDiscrete(scaleType)) {
-        if (config.range && config.range.ordinal) {
-          return 'ordinal';
+        if (config.range && isVgScheme(config.range.ordinal)) {
```

Comment:
I still don't understand why do we need to read config.range at all?  

Unless you have convincing reason, maybe we should just return 

```
{scheme: 'blues', count};
```

Provide that count use the same rule as size above. (Write a function for `defaultContinuousToDiscreteCount()`)

Reply:
If the users specify different scheme in `config.range.ordinal`, should we use the specified scheme instead of `blues`?